exercises/css-canvas-medical/styles.css

```diff
@@ -0,0 +1,243 @@
+  display: flex;
+}
+
+.head-link {
+  display: flex;
+  font-family: 'Avenir', sans-serif;
+  margin-top: 5%;
+  margin-right: 3%;
+  margin-left: 3%;
+  color: #a6a6a6;
+  font-size: 21px;
+}
+
+.head-contact {
+  margin-top: 2.5%;
+  border: 1px solid #294363;
+  color: white;
+  background-color: #294363;
+  font-family: 'Avenir', sans-serif;
+  border-radius: 5px;
+  font-size: 21px;
+  padding: 2% 3% 2% 3%;
+}
+
+.container-img-margin {
+  width: 85%;
+  margin: auto;
+  border-bottom: 1px lightgray solid;
+  background-image: url(images/hero.jpg);
+  background-size: cover;
+  height: 650px;
+  margin-top: 75px;
+}
+
+.img-tint {
+  background-color: rgba(0, 0, 0, 0.25);
+  width: 100%;
+  height: 100%;
+}
+
+.img-head {
+  color: white;
+  font-family: 'Tiempos Headline', serif;
+  font-size: 20px;
+  letter-spacing: 1px;
+  margin-top: 15%;
+  margin-left: 15%;
+}
+
+.img-text {
+  color: white;
+  font-family: 'Avenir', sans-serif;
+  line-height: 2;
+  font-size: 16px;
+  margin-left: 15%;
+}
+
+.img-link {
+  margin-top: 10%;
+}
+
+.get-started {
+  border: solid 1px #294363;
+  color: white;
+  font-family: 'Avenir', sans-serif;
+  font-size: 24px;
+  margin-left: 15%;
+  background-color: #294363;
+  padding: 15px 50px 15px 50px;
+  border-radius: 5px;
+}
+
+.container-city {
+  border-bottom: rgb(230, 230, 230) 1px solid;
+}
+
+.container-city-2 {
+  width: 100%;
+  margin-top: 10%;
+}
+
+.city-head {
+  text-align: center;
+  font-family: 'Tiempos Headline', serif;
+  color: #6e6e6e;
+  font-size: 18px;
+}
+
+.city-text {
+  text-align: center;
+  font-family: 'Avenir', sans-serif;
+  color: #6e6e6e;
+  font-size: 18px;
+  line-height: 2;
+}
+
+.city-city {
+  font-family: 'Avenir', sans-serif;
+  color: #6e6e6e;
+  margin-top: 10%;
+  line-height: 2.5;
+  text-align: center;
+  margin-bottom: 25%;
+}
+
+.start-head {
+  font-family: 'Tiempos Headline', serif;
+  color: #6e6e6e;
+  margin-top: 10%;
+  font-size: 20px;
+}
+
+.start-text {
+  font-family: 'Avenir', sans-serif;
+  color: #6e6e6e;
+  font-size: 20px;
+  margin-bottom: 20px;
+}
+
+.get-started-bottom {
+  border: solid 1px #294363;
+  color: white;
+  font-family: 'Avenir', sans-serif;
+  font-size: 24px;
+  background-color: #294363;
+  border-radius: 5px;
+  padding: 15px 50px 15px 50px;
+}
+
+.start-link {
+  text-align: end;
+  margin-top: 15%;
+}
+
+.container-start {
+  border-bottom: #6e6e6e 1px solid;
+}
+
+.start {
+  margin-bottom: 10%;
+}
+
+.container-end {
+  background-color: #525351;
+}
+
+.end-text {
+  font-family: 'Avenir', sans-serif;
+  color: white;
+  line-height: 1.6;
+  margin-top: 10%;
+  font-size: 20px;
+}
+
+.link-end {
+  font-family: 'Avenir', sans-serif;
+  color: white;
+  font-size: 24px;
+  line-height: 2.5;
+}
+
+.end-link-head {
+  font-family: 'Avenir', sans-serif;
+  color: white;
+}
+
+#canvas-white {
+  width: 25%;
+  margin-top: 20%;
+}
+
+.col-1-2-end {
+  width: 50%;
+  margin-top: 15%;
+  margin-bottom: 15%;
```

Comment:
Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.